### PR TITLE
Fix Dependencies

### PR DESCRIPTION
**Pull request by Kevan-Y**
Fixes [#2678](https://github.com/Seneca-CDOT/telescope/issues/2678)
 
 ### Description
  
* Remove elastic-apm-node
* Remove @elastic/ecs-pino-format
* Update Jest (To fix deprecated dependencies)

### Test
image

### CODE REVIEW

**Comment by Kevan-Y** [comment]
https://github.com/Seneca-CDOT/satellite/blob/b1d93348abd09e03d4509fda8fef9545c99d288a/README.md?plain=1#L19-L32
JWT is not part of Elastic APM, right?

**Comment by humphd** [comment]
Correct, that stuff is unrelated to the APM bits.

File: src/logger.js [hunk 1]
```diff
@@ -1,15 +1,9 @@
...
-    pino(require('@elastic/ecs-pino-format')({ convertReqRes: true }))
-  : // Pretty debug logging
-    pino({
-      prettyPrint: {},
-      prettifier: require('pino-colada'),
-    });
+// Pretty debug logging
+let logger = pino({
+  prettyPrint: {},
+  prettifier: require('pino-colada'),
```
Comment (humphd) [inline src/logger.js:6]: In a follow-up, I'd like to fix how we do this.  `pino-collada` is great for debugging, but it's not what we should use in production.  Maybe you can work on that next?

**Review by humphd — APPROVED** [review]

File: src/logger.js (hunk 1 above)
Comment (Kevan-Y) [reply src/logger.js:6]: Yes, will do, since I'm also working with pino in [#2499](https://github.com/Seneca-CDOT/telescope/issues/2499). I will fix there

**Review by humphd — APPROVED** [review]

**Review by chrispinkney — APPROVED** [review]

**Comment by humphd** [comment]
@Kevan-Y go ahead and merge this when you're ready.

**Pull request merged**